### PR TITLE
[skip gpuci] Feat/cibuildwheel

### PR DESCRIPTION
**Pull request by sevagh**
Build pip wheels from GitHub Actions.

### CODE REVIEW

File: .github/workflows/wheels.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@ jobs:
 
       test-container-amd64: "rapidsai/manylinux_2_27"
       test-container-arm64: "rapidsai/manylinux_2_31" 
+      test-docker-options: "-cap-add CAP_SYS_PTRACE --shm-size=8g"
```
Comment (wence-) [inline .github/workflows/wheels.yml]:
```suggestion
      test-docker-options: "--cap-add CAP_SYS_PTRACE --shm-size=8g"
```

File: .github/workflows/wheels.yml [hunk 2]
```diff
@@ -61,6 +62,7 @@ jobs:
...
```
Comment (wence-) [inline .github/workflows/wheels.yml]:
```suggestion
      test-docker-options: "--cap-add CAP_SYS_PTRACE --shm-size=8g"
```

**Review by wence- — COMMENTED** [review]
Minor fix to docker command line

File: setup.py [hunk 3]
```diff
@@ -69,21 +69,23 @@ def get_ucp_version():
...
+extras_require = {
+    "test": [
+        "pytest",
+        "pytest-asyncio",
+    ]
+}
 
 setup(
-    name="ucx-py",
+    name="ucx-py"+os.getenv("RAPIDS_PY_WHEEL_CUDA_SUFFIX", default=''),
```
Comment (jakirkham) [inline setup.py:80]: Wonder if we could capture this another way. Perhaps like a common dependency that all packages depends on?

**Comment by sevagh** [comment]
Superseded by #901

**Pull request closed without merging**